src/main/plugins/discordrpc.ts

```diff
@@ -74,6 +74,12 @@ export default class DiscordRichPresence {
             this.disconnect()
         });
 
+        // If Discord is closed, allow reconnecting
+        this._client.transport.once('close', () => {
```

Comment:
I used client.transport.once('close') cause client.once('disconnect') seems to only be emitted durring the connection phase

Reply:
Ah, figured as much.